libraries/FFGAppImport/AssetImport/FetchContent.cs

```diff
@@ -295,19 +344,14 @@ private void ExportTexture(AssetPreloadData asset)
                 throw new ArgumentNullException("asset");
             Texture2D texture2D = new Texture2D(asset, false);
             texture2D = new Texture2D(asset, true);
-            Directory.CreateDirectory(contentPath);
-            Directory.CreateDirectory(contentPath + "/img");
+            string imgPath = Path.Combine(contentPath, "img");
+            Directory.CreateDirectory(imgPath);
             // Default file name
-            string fileCandidate = contentPath + "/img/" + asset.Text;
-            string fileName = fileCandidate + asset.extension;
+            string fileCandidate = Path.Combine(imgPath, asset.Text);
+            // This should apends a postfix to the name to avoid collisions
+            string fileName = GetAvailableFileName(fileCandidate, asset.extension);

```

Comment:
This problem is more complicated.  If we have a collision, how do we ensure we are getting the right one?  What if the packages change in the future?  The way to deal with this is either not allow collisions, or resolve it with information about the source of the data so that it is consistent.

Reply:
Default filenames may also be a subject to change. In those cases, we have to change the ini files to deal with the changes and raise the minimum suported version.
Adding the level like level3 to the filename (on collision) is a good suggestion. If ffg adds an image to a level, it wouldn't cause all other generated image files to be incremented.
We could do hashes, fingerprints or other heuristics but this would add alot more complexion. I'd say avoid that.